SimTracker/SiPhase2Digitizer/plugins/PixelBrickedDigitizerAlgorithm.h

```diff
@@ -0,0 +1,26 @@
+
+#ifndef _SimTracker_SiPhase2Digitizer_PixelBrickedDigitizerAlgorithm_h
+#define _SimTracker_SiPhase2Digitizer_PixelBrickedDigitizerAlgorithm_h
```

Comment:
Since this is some new code, you can merge this in the .cc file, as it is more convenient for plugins whose header file shouldn't get linked anywhere

Reply:
Do you mean only for the PixelBrickedDigitizerAlgorithm? All the available algorithms in the plugin area have their header files there.

Reply:
@suchandradutta I was referring to this one in particular, because it is some new code and it can be coded as such since the beginning. Migration of already existing plugins is welcome, but certainly not intended for this PR

Reply:
@perrotta  All these individual algorithms are created/used in the digitizer(Phase2TrackerDigitizer) itself. The header of these individual algorithms are used there. So I am afraid I can not avoid the separate header files.

Reply:
> @perrotta All these individual algorithms are created/used in the digitizer(Phase2TrackerDigitizer) itself. The header of these individual algorithms are used there. So I am afraid I can not avoid the separate header files.

Correct! Sorry for the noise, please forget this comment then